### ISSUE TITLE
view-uniform can't be found when running from budo

### ISSUE THREAD

**Issue by Erkaman** [mikolalysenko/regl#28]
None of the examples that uses  `view`-uniforms seems to work anymore, when running from budo. If I for example run

```
budo example/camera.js -t glslify --live --open
```

I just get a black screen, and the error message

```
check.js:21Error: (regl) invalid uniform view in command
check.js:20Uncaught Error: (regl) invalid uniform view in command
```

the same thing happens in all the examples that uses a view-uniform. I used `git-bisect` to determine that the error was introduced in commit  e9307158b6 . But that makes no sense, because it only changes a string. 

Anyways, If I here:

```javascript
      check.optional(function () {
        function check (pred) {
          env.assert(scope, pred, 'invalid or missing uniform "' + name + '"')
        }
```

comment out the line 

```javascript
env.assert(scope, pred, 'invalid or missing uniform "' + name + '"')
```

All the examples work again. 

If I build the gallery, then all the examples work. But from budo, they don't work. And since it takes forever to build the gallery, it would be really convenient if I could run them from budo.

**Comment by mikolalysenko** [comment]
Same problem as this https://github.com/mikolalysenko/regl/issues/23.  That Array.isArray check should also check if it is a typedarray.  I'll fix it ASAP

**Comment by mikolalysenko** [comment]
Solved in PR #31
